atlassian_jwt_auth/contrib/flask_app/__init__.py

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
@dbaxa So, the main issue that this naming strategy is trying to fix is the clobbering of core packages like `flask`, `django`, and `requests` that prevent sibling packages under `contrib` from using those libraries.

Not many libraries have this problem where they include stubbing for Django, Flask, etc, but the most notable, `python-social-auth` which contains the "Login with X" code for almost every cloud service and it follows the pattern of `django_app`, `flask_app`, etc. as you see [here](https://github.com/omab/python-social-auth/tree/master/social/apps). I think its a good and valuable pattern in this scenario.